tests/test_cmd.py

```diff
@@ -19,6 +19,7 @@ def cmd_convert(model_name):
 def test_cmd_convert():
     MODEL_LIST = (
         'abs.float32',
+        'abs-sqrt.float32',
```

Comment:
In general, we don't need to add more operator tests to the command line tool test, but won't block the PR merging.

Reply:
https://github.com/jackwish/tflite2onnx/pull/37/commits/90c439701cc1ea57daf7cf540d8b0033410eb565#diff-60497cc703a3a9baddaf30a3384a7774b89bc52007e28c34828758f055fcd72cL22